src/main/java/org/jboss/tools/intellij/openshift/ui/component/CreateComponentDialogStep.java

```diff
@@ -79,20 +71,11 @@ protected void textChanged(DocumentEvent e) {
                 updateState();
             }
         });
-        sourceTypeComboBox.addItemListener(e -> {
-            if (e.getStateChange() == ItemEvent.SELECTED) {
-                model.setSourceType((ComponentSourceType) sourceTypeComboBox.getSelectedItem());
-                updateState();
-            }
-        });
         contextTextField.getDocument().addDocumentListener(new DocumentAdapter() {
             @Override
             protected void textChanged(DocumentEvent e) {
                 model.setContext(contextTextField.getText());
                 informationLabel.setVisible(model.isProjectHasDevfile() || model.isProjectIsEmpty());
-                componentTypeTree.setEnabled(!model.isProjectHasDevfile());
```

Comment:
We should do the same on the list when local project has devfile, no ?

Reply:
good catch !